src/assets/css/_main.scss

```diff
@@ -1661,7 +1663,7 @@ a.tile-block-link {
     transform: rotate(360deg);
   }
 }
-
+/* stylelint-enable  selector-id-pattern */
```

Comment:
I hate doing this, but there's just too many files for it to make sense to roll in here. Filed https://github.com/mozilla/extension-workshop/issues/1219 to come back and address.